frontend/src/components/lease/List.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'react-i18next';
+import { Lease } from '../../lib/k8s/lease';
+import { SectionBox, SectionFilterHeader } from '../common';
+import ResourceTable from '../common/Resource/ResourceTable';
+
+export function LeaseList() {
+  const { t } = useTranslation(['glossary', 'leases']);
```

Comment:
Let's use `resource` instead of `leases`.

Reply:
fixed